test/unit/ui/app/actions.spec.js

```diff
@@ -407,22 +409,181 @@ describe('Actions', () => {
 
   describe('#addNewAccount', () => {
 
-    let addNewAccountSpy
-
-    afterEach(() => {
-      addNewAccountSpy.restore()
-    })
-
     it('Adds a new account', () => {
       const store = mockStore({ metamask: devState })
 
-      addNewAccountSpy = sinon.spy(background, 'addNewAccount')
+      const addNewAccountSpy = sinon.spy(background, 'addNewAccount')
 
       return store.dispatch(actions.addNewAccount())
         .then(() => {
           assert(addNewAccountSpy.calledOnce)
         })
     })
+
+  })
+
+  describe('#checkHardwareStatus', () => {
+
+    let checkHardwareStatusSpy
+
+    beforeEach(() => {
+      checkHardwareStatusSpy = sinon.stub(background, 'checkHardwareStatus')
+    })
+
+    afterEach(() => {
+      checkHardwareStatusSpy.restore()
+    })
+
+    it('calls checkHardwareStatus in background', (done) => {
+
+      const store = mockStore()
+
+      store.dispatch(actions.checkHardwareStatus('ledger', `m/44'/60'/0'/0`))
+      assert.equal(checkHardwareStatusSpy.calledOnce, true)
+      done()
```

Comment:
Nit: It doesn't seem like this method needs to use `done` - it appears to be synchronous.